### ISSUE TITLE
Create a PPXML reader

### ISSUE THREAD

**Issue by pkiraly** [pkiraly/qa-catalogue#386]
http://format.gbv.de/pica/ppxml:
"PPXML ist eine an der Deutschen Nationalbibliothek verwendete Kodierung von [PICA+](http://format.gbv.de/pica) in [XML](http://format.gbv.de/xml). Eine Besonderheit von PPXML gegenüber den anderen PICA-Serialisierungen ist die explizite Trennung von bibliographischer (<global>), lokaler (<local>) und Exemplarebene (<copy>)."

Deutsche Nationalbibliothek supports PPXML format in their SRU interface (https://services.dnb.de/sru/dnb?version=1.1&operation=searchRetrieve&query=sgt%3D???%20NOT%20ido%3D785055750&recordSchema=PicaPlus-xml).

**Comment by nichtich** [comment]
I don't think this is needed. There are tools to convert between PICA serializations. Support of normalized PICA as default and Plain Pica for human-readable small datasets should be enough.
